ssz-rs/src/list.rs

```diff
@@ -12,7 +12,7 @@ use crate::{
 #[cfg(feature = "serde")]
 use serde::ser::SerializeSeq;
 #[cfg(feature = "serde")]
-use std::marker::PhantomData;
+use core::marker::PhantomData;
```

Comment:
can we follow the other types here and move `PhantomData` to the `crate::lib` and export one or the other depending on std or not?